core/src/test/scala/org/apache/spark/sql/delta/DeltaProtocolVersionSuite.scala

```diff
@@ -1209,6 +1209,39 @@ trait DeltaProtocolVersionSuiteBase extends QueryTest
       Protocol(TABLE_FEATURES_MIN_READER_VERSION, TABLE_FEATURES_MIN_WRITER_VERSION)
         .withFeature(TestReaderWriterMetadataNoAutoUpdateFeature)))
 
+  testCreateTable(
+    name = "feature with a dependency",
+    props = Map(
+      DeltaConfigs.MIN_READER_VERSION.key -> TABLE_FEATURES_MIN_READER_VERSION.toString,
+      DeltaConfigs.MIN_WRITER_VERSION.key -> TABLE_FEATURES_MIN_WRITER_VERSION.toString,
+      s"delta.feature.${TestFeatureWithDependency.name}" -> "enabled"),
```

Comment:
```suggestion
      s"delta.feature.${TestFeatureWithDependency.name}" -> "supported"),
```
Here and below 😀

Reply:
I copied this from the test cases above this one. I changed the tests that I've added to use "supported".